src/EditorFeatures/TestUtilities/StubVsServiceExporter`1.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests;
+
+[Export(typeof(IVsService<>))]
```

Comment:
Not sure I knew this works -- I suspect I can do much more evil in the future now.